### PR TITLE
Migrating to setup.cfg

### LINKED ISSUE #294: Use a setup.cfg
Discussions about the future PR using setup.cfg

### PR DESCRIPTION
**Pull request by victorbnl**
Closes #294

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,58 +1,16 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```
Comment (claudep) [inline setup.py:2]: With Python 3, this line should not be needed.

File: setup.py [hunk 2]
```diff
@@ -1,58 +1,16 @@
...
-        'Programming Language :: Python :: 3.9',
-        'Programming Language :: Python :: 3 :: Only',
-        'Topic :: Documentation',
-        'Topic :: Utilities',
-        'Topic :: Printing',
-        'Topic :: Multimedia :: Graphics :: Graphics Conversion',
-        'Topic :: Software Development :: Libraries :: Python Modules',
-        'Topic :: Text Processing :: Markup :: XML',
-    ],
+    setup_requires=[]+pytest_runner
```
Comment (claudep) [inline setup.py:15]: Please add spaces around `+`.

**Review by claudep — COMMENTED** [review]
Thanks, small changes required (and rebasing would be nice).

**Comment by victorbnl** [comment]
Did you mean rebasing for not having these two commits above?

**Comment by claudep** [comment]
I asked the rebase because I committed a tweak to be able to know which flag is failing to download in the test suite.

**Comment by claudep** [comment]
@deeplook  was right about the Afghanistan flag! Anyway, this has nothing to do with this patch, we can go ahead.

**Comment by deeplook** [comment]
Using better tooling like in https://github.com/deeplook/svglib/pull/296 will make this type of contributions hopefully superfluous. ;)

**Pull request merged**